contracts/DixelClubV2Factory.sol

```diff
@@ -47,20 +47,21 @@ contract DixelClubV2Factory is Constants, Ownable {
     function createCollection(
         string calldata name,
         string calldata symbol,
+        string calldata description,
```

Comment:
Could you explain how separating description from MetaData struct can save SLOAD?

Reply:
The description causes overhead in the off-chain. However, metadata is often read on-chain, and if the description length exceeds 256 bits, multiple 'sload's occur to read metadata on on-chain logic.

Even if you are not worried about this, it is recommended to resize the metadata itself to be read as a single 'sload' because it is loaded in the on-chain logic.

Reply:
fyi evm performs sload and sstore every 256 bits!